### PR TITLE
feat: support nested property in titleField

### PR DESCRIPTION
**Pull request by arthurdenner**
Hi! 👋🏽 

Thank you for this nice plugin and CMS! 😃

This PR adds support for nested properties in the `titleField` property using [dlv](https://github.com/developit/dlv).

Also, it updates the `kind` prop for the `Cancel` button - the resulting UI is the same.

Let me know if you have any suggestions or objections to this change.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +26,6 @@
   },
   "peerDependencies": {
     "@sanity/base": "^2.0.9",
-    "@sanity/react-hooks": "^2.0.9",
```
Comment (arthurdenner) [inline package.json:28]: Removed because it's a duplicate. The same peerDependency is declared at line 31.

**Comment by sjelfull** [comment]
@runeb Looks good to me. WDYT?

**Comment by runeb** [comment]
Thank you for the patch @arthurdenner ! Looks good to me. Would be nice to add usage example of this to README in config so people are aware of it.

**Review by runeb — APPROVED** [review]

**Comment by arthurdenner** [comment]
Thank you for the suggestion, @runeb. I've added a comment highlighting the support.

Let me know if you prefer a complete JSON example instead.

**Comment by sjelfull** [comment]
@arthurdenner Thanks, that is good enough

**Comment by arthurdenner** [comment]
Sorry for bothering you, but is there any ETA for the release of this change? 😊

**Comment by sjelfull** [comment]
@arthurdenner Thanks for the patience, it should be out now.

**Pull request merged**